src/hats_import/catalog/resume_plan.py

```diff
@@ -239,15 +239,15 @@ def read_histogram(self, healpix_order, which_histogram: str = "row_count"):
                 aggregate_histogram.add(partial)
 
             file_name = file_io.append_paths_to_pointer(self.tmp_path, histogram_binary_file)
-            with open(file_name, "wb+") as file_handle:
+            with open(str(file_name), "wb+") as file_handle:
```

Comment:
This is going to have issues with non-local file systems. See similar changes in https://github.com/astronomy-commons/hats-import/pull/620

```suggestion
            with file_name.open("wb+") as file_handle:
```